lib/widgets/icons.dart

```diff
@@ -48,65 +48,77 @@ abstract final class ZulipIcons {
   /// The Zulip custom icon "clock".
   static const IconData clock = IconData(0xf108, fontFamily: "Zulip Icons");
 
+  /// The Zulip custom icon "contacts".
+  static const IconData contacts = IconData(0xf109, fontFamily: "Zulip Icons");
```

Comment:
nit:

    nav: Add icons from Figma
    
    The icons are taken from
      https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=544-22131&node-type=canvas&t=f4vYPq2CZUpFuf7V-0.

Write instead:

    icons: Add icons for bottom nav bar
    
    The icons are taken from Figma:
      https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=544-22131&node-type=canvas&t=f4vYPq2CZUpFuf7V-0

Avoiding `.` there avoids ambiguity about what the URL is.

The summary line can be more specific than "nav" about what the icons are; there are other icons involved in nav. And our usual prefix for editing icons is "icons:".